share/locale/mscore_nl.ts

```diff
@@ -3441,12 +3441,12 @@ space unit</extracomment>
     <message>
         <location filename="../../fluid/fluid_gui.ui" line="47"/>
         <source>Move SoundFont up</source>
-        <translation>Verplaats SoundDont omhoog</translation>
+        <translation>Verplaats SoundFont omhoog</translation>
```

Comment:
@jeetee please take a look at this on Transifex.

Reply:
Seems Ad Stakenborg  already fixed that 12d ago (or even 9m ago for that specific one?)